keptn/keptn
# feat: Add OAuth scopes to swagger and add possibility to disable deprecated APIs

## This PR

introduces basic documentation about RBAC scopes planned for each API endpoint. 
Furthermore, it allows configuring if the rendering for deprecated endpoints should be rendered or not. 

### Related Issues


Fixes #7352

### How to test

Run:
- `helm template keptn .\installer\manifests\keptn --set control-plane.features.oauth.prefix="myPrefix:" --set control-plane.features.oauth.enabled=true  > out.yaml` 
- `helm template keptn .\installer\manifests\keptn --set control-plane.features.oauth.prefix="keptn:" --set control-plane.bridge.oauth.enabled=true  > out2.yaml`
 
and verify the output file are correct.
